config/sync/editor.editor.full_html.yml

```diff
@@ -1,75 +1,124 @@
-uuid: 749217f1-edf8-4bbb-850f-742ad8637ada
+uuid: 7e6b08b3-84c7-4473-9edd-499b7cf7c807
 langcode: en
 status: true
 dependencies:
   config:
     - filter.format.full_html
   module:
-    - ckeditor
-_core:
-  default_config_hash: omGUBeXljcaV1QHmUMrbBSFotgOdUg33ZI2mUItpSVE
+    - ckeditor5
 format: full_html
-editor: ckeditor
+editor: ckeditor5
 settings:
   toolbar:
-    rows:
-      -
-        -
-          name: Formatting
-          items:
-            - Bold
-            - Italic
-            - Strike
-            - Superscript
-            - Subscript
-            - '-'
-            - RemoveFormat
-        -
-          name: Links
-          items:
-            - DrupalLink
-            - DrupalUnlink
-        -
-          name: Lists
-          items:
-            - BulletedList
-            - NumberedList
-        -
-          name: Media
-          items:
-            - Blockquote
-            - DrupalImage
-            - video_embed
-            - Table
-            - HorizontalRule
-            - node
-        -
-          name: 'Block Formatting'
-          items:
-            - Format
-            - Styles
-        -
-          name: Tools
-          items:
-            - ShowBlocks
-            - Source
+    items:
+      - bold
+      - italic
+      - strikethrough
+      - superscript
+      - subscript
+      - removeFormat
+      - '|'
+      - link
+      - '|'
+      - bulletedList
+      - numberedList
+      - '|'
+      - blockQuote
+      - drupalInsertImage
+      - mediaembed
+      - insertTable
+      - embeddedContent
+      - node
+      - horizontalLine
+      - '|'
+      - heading
+      - codeBlock
+      - style
+      - '|'
+      - showBlocks
+      - sourceEditing
   plugins:
-    stylescombo:
-      styles: "span.damo-silver|Silver\r\nspan.damo-blue|Blue\r\nspan.damo-orange|Orange\r\nspan.damo-white|White"
-    language:
-      language_list: un
-    video_embed:
-      defaults:
-        children:
-          autoplay: true
-          responsive: true
-          width: '854'
-          height: '480'
+    ckeditor5_codeBlock:
+      languages:
+        -
+          label: 'Plain text'
+          language: plaintext
+        -
+          label: C
```

Comment:
biztos kellenek ezek a tipusok. Tippre eddig csak a <pre>t hasnzaltuk, ezt a CKEditor5 code-ra allitja, de akkor is eleg a Plain text tipus, a tobbit nem kell engedelyezni